client/protocol.go

```diff
@@ -129,10 +130,28 @@ func fastFetchCluster(addrs []string, proposeTimeout time.Duration) (*curpapi.Fe
 
 // Propose the request to servers, if use_fast_path is false, it will wait for the synced index
 func (c *protocolClient) Propose(cmd *xlineapi.Command, useFastPath bool) (*proposeRes, error) {
-	if useFastPath {
-		return c.fastPath(cmd)
+	var res *proposeRes
+	var err error
+	for {
+		if useFastPath {
+			res, err = c.fastPath(cmd)
+		} else {
+			res, err = c.slowPath(cmd)
+		}
+
+		if errors.Is(err, errors.New("wrong cluster version")) {
```

Comment:
make a const `wrong cluster version` error might be better

Reply:
You have right, and I've made the changes you suggested.